example/javascriptlib/basic/6-client-server-realistic/build.mill

```diff
@@ -0,0 +1,60 @@
+package build
+import mill._, javascriptlib._
+
+object server extends JestModule {
+  def npmDeps =
+    Seq("@types/cors@^2.8.17", "@types/express@^5.0.0", "cors@^2.8.5", "express@^4.21.1")
+
+  def npmDevDeps =
+    super.npmDevDeps() ++ Seq("@types/supertest@^6.0.2", "supertest@^7.0.0")
+
+  override def testConfigSource = Task.Source(millSourcePath / "jest.config.ts")
+
+  override def bundleFlags = Map("external" -> Seq("express", "cors"))
+
+}
+
+object client extends RsWithServeModule {
+  override def mkENV = Task {
+    super.mkENV() ++ Map("PORT" -> "3000")
+  }
+}
+
+// Documentation for mill.example.javascriptlib
+
+/** Usage
```

Comment:
Going to `localhost:3001` after `server.run` gives me a `Cannot GET /` error. How do you use this todomvc app?

Also, why is the client configured to port 3000 when the server seems to listen on port 3001?

Reply:
that may be the reason for the cannot GET / error - a misconfiguration.
Will investigate and fix

Reply:
might be a misconfiguration, will investigate and fix